src/components/SuperDrawer/SuperDrawer.stories.tsx

```diff
@@ -368,55 +377,70 @@ function TestDrawerContent({ book, hasActions = true }: TestDrawerContentProps)
   }
 
   return (
-    <SuperDrawerContent
-      actions={
-        hasActions
-          ? [
-              { label: "Close", onClick: () => closeDrawer(), variant: "tertiary" },
-              { label: `Purchase "${book.bookTitle}"`, onClick: handlePurchase },
-            ]
-          : undefined
-      }
-    >
-      <h2 css={Css.xlSb.mb1.$}>{book.bookTitle}</h2>
-      <p css={Css.base.$}>
-        <strong>Description:</strong> {book.bookDescription}
-      </p>
-      <hr css={Css.my1.$} />
-      <div css={Css.df.gap1.aic.$}>
-        <p>
-          <strong>Author:</strong> {book.authorName}
+    <>
+      <SuperDrawerHeader hideControls={hideControls}>
```

Comment:
Nice! I think this is a much better approach. The previous API was definitely the right one to start with, but now that we're being asked to "do more" / keep it up to date/etc.